src/examples/CSV_README.md

```diff
@@ -0,0 +1,268 @@
+# Fast C++ CSV Parser
```

Comment:
Yeah now that I understand the changes you made I'd keep this next to `csv.h`. Fine to leave the name as is